README.md

```diff
@@ -1 +1,23 @@
-# COVID_moonshot_data_clean
\ No newline at end of file
+# COVID_moonshot_data_clean
+This repository contains cleaned versions of the data in
+https://github.com/asapdiscovery/COVID_moonshot_submissions.
+
+## Files
+* `cdd_noncovalent_dates_2023_10_18_unfiltered_clean.csv`: All noncovalent available
+data as of October 10, 2023, cleaned to remove data without a noted SMILES string or
+IC50 value
```

Comment:
Does this mean numerical IC50, or does this include measurements below or above detectable range?

What was the original source of the data before cleaning? CDD Vault?

Do you mean "filtered" instead of "cleaned"?

Is the data indexed by suspected_SMILES?

Have we documented what all the fields mean?

Did we note this was from the fluorescence assay and cite the experimental assay description in the preprint?

Reply:
1. these are all semi-quantitative data (I can make a note of that in the README). I did include the `In_range` field that lets the user know if a given measurement was below, above, or within the assay range
2. this is all from the CDD vault. I can make a note of that as well
3. for this dataset, I've removed all data that doesn't have a SMILES or IC50 value, but have done no other filtering. to me, that's "cleaning", but if "cleaning" typically implies that no data points were removed, I can change the wording to "filtered"
4. what do you mean by indexed? the first column for both CSV files is the PostEra ID, but I can rearrange the files so SMILES is first
5. yes, the next section in the README explains the column headers
6. I have not, but I'll include that in the README

Reply:
* Can you modify the README to clarify these points?
* I'd use "filtering", since "cleaning" to me implies changing data rather than just omitting some data.
* Re "indexed": The first column in the file should be the SMILES we want people to use (suspected_SMILES), and the second column should be the title (PostEra ID). This is how OpenEye and other readers expect the data to be arranged.